src/app/components/select-items/select-items.component.spec.ts

```diff
@@ -13,11 +14,38 @@ describe('SelectItemsComponent', () => {
       .compileComponents();
 
     fixture = TestBed.createComponent(SelectItemsComponent);
+    fixture.componentRef.setInput('possibleItemNames', possibleItemNames);
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should add empty item', () => {
+    component.addItem();
+    expect(component.getSelectedItem()).toBe('');
+  });
```

Comment:
This test is unclear to me. Could you share why do we want to test `should add empty item`?